.github/workflows/check-pr.yml

```diff
@@ -27,11 +27,15 @@ jobs:
     name: Run tests
     needs: deploy
     uses: ./.github/workflows/run-integration-tests.yml
+    permissions:
```

Comment:
Just sense checking what these added permissions are for?

Reply:
The permissions are needed to be able to get the required parameters and outputs from the stack without updating it in the template.yaml